ts/parts/Pointer.ts

```diff
@@ -637,16 +637,53 @@ Highcharts.Pointer.prototype = {
         shared: (boolean|undefined),
         e?: Highcharts.PointerEventObject
     ): Highcharts.PointerHoverDataObject {
-        var hoverPoint: Highcharts.Point,
+        var chart = this.chart,
+            hoverPoint: Highcharts.Point,
             hoverPoints = [] as Array<Highcharts.Point>,
             hoverSeries = existingHoverSeries,
             useExisting = !!(isDirectTouch && existingHoverPoint),
             notSticky = hoverSeries && !hoverSeries.stickyTracking,
+            isInsidePane = (
```

Comment:
As above, we shouldn't pollute core Pointer stuff with polar features. Instead, write an event hook and extend it from the Polar.ts file.